Assignment01/step-by-step-java.md

```diff
@@ -15,23 +15,23 @@ As a reminder, this is how the services will interact with each other:
 
 ## Step 1. Run the VehicleRegistration service
 
-1. Open the `java` folder in VS Code. This guide assumes VS Code, but feel free to use an editor or IDE you're comfortable with.
+1. Open the source-code folder in VS Code. This guide assumes VS Code, but feel free to use an editor or IDE you're comfortable with.
```

Comment:
I think (but I'm not 100% sure) the preferred writing is **source code** (with a space rather than a dash).